tests/test_discovery.py

```diff
@@ -1,33 +1,28 @@
-import re
 
-from tap_tester import menagerie, connections
-
-from base import TestLinkedinAdsBase
-
-class LinkedinAdsDiscoveryTest(TestLinkedinAdsBase):
-
-    def name(self):
```

Comment:
Any specific reason to remove the class from here?

Reply:
Removed by mistake. No perticular reason.